inst/scripts/new_post.R

```diff
@@ -66,14 +72,15 @@ local({
           input$title, author = input$author, ext = input$format,
           categories = input$cat, tags = input$tag,
           file = gsub('[-[:space:]]+', '-', input$file),
-          slug = input$slug, subdir = input$subdir, date = input$date
+          slug = input$slug, subdir = input$subdir, date = input$date,
+          kind = input$kind
         )
         shiny::stopApp()
       })
       shiny::observeEvent(input$cancel, {
         shiny::stopApp()
       })
     },
-    stopOnCancel = FALSE, viewer = shiny::dialogViewer('New Post', height = 500)
+    stopOnCancel = FALSE, viewer = shiny::dialogViewer('New Post', height = 570)
```

Comment:
Could you try to move the `Archetype` menu to the right of `Tags` and see how it looks like? I prefer not to increase the height of this addin if possible.